### ISSUE TITLE
Bounce Rate Graph Display Concerns

### ISSUE THREAD

**Issue by nijamaDev** [benvinegar/counterscale#128]
Hi, I'm unsure whether the Bounce Rate graph is displaying correctly; it seems a bit odd. 

The image shows that the bounce rate for the last day is at 46%, but the graph rises to 46% of the graph's viewable area.

I believe the line should only rise to 46% of the height of the views (or visitors, not sure) for that specific day.

![image](https://github.com/user-attachments/assets/322deca9-30c6-4ef9-93de-4eb3157b0b99)

**Comment by ka2590** [comment]
> P.S.: In the tooltip, 'Views' should come before 'Visitors' since the number of views will always be higher.

I disagree. It is logical to say "X number of visitors viewed X number of pages and X number of those visitors bounced".

**Comment by nijamaDev** [comment]
> > P.S.: In the tooltip, 'Views' should come before 'Visitors' since the number of views will always be higher.
> 
> I disagree. It is logical to say "X number of visitors viewed X number of pages and X number of those visitors bounced".

I see your point, but I prefer to phrase it like this:

_"There were X views from X visitors, with an X% bounce rate."_

 I think it sounds a bit clearer without repeating 'visitors' twice. What do you think?

What I mean in the P.S. is that views are always higher on the graph, as the views are equal or greater than the visitors. So, the tooltip could follow this order by listing views before visitors.
